website/content/en/v0.16.1/AWS/provisioning.md

```diff
@@ -206,13 +206,13 @@ spec:
 
 ### Block Device Mappings

```

Comment:
Just realized this change should also be propagated to the preview documentation on path `website/content/en/preview/AWS/provisioning.md`